### ISSUE TITLE
withStyles() results in exception due to _processStyle not returning a promise

### ISSUE THREAD

**Issue by cblades-tc** [ficusjs/ficusjs#1]
When using withStyles(), an exception is raised:

```
ficusjs.js:259 Uncaught TypeError: Cannot read property 'then' of undefined
    at HTMLElement._injectStyles (ficusjs.js:259)
    at HTMLElement.created (ficusjs.js:249)
    at HTMLElement._init (ficusjs.js:60)
    at HTMLElement._checkInit (ficusjs.js:36)
    at HTMLElement.connectedCallback (ficusjs.js:24)
    at createComponent (ficusjs.js:16)
    at index.html?_ijt=docne8ki0eg2au6o94jl5r6ta5:22
```

The issue appears to be that _injectStyles expects _processStyle to return a promise:
```
          this._processStyle(styleItems)
            .then(cssToImport => this._createAndInjectStylesheet(cssToImport, { 'data-tag': this.componentTagName }))
```

**Comment by cblades-tc** [comment]
If you accept pull requests, this seems to fix it: https://github.com/cblades-tc/ficusjs/commit/ec6bcad5da74fca963184052777c9f207be3e206

**Comment by ducksoupdev** [comment]
@cblades-tc thanks for letting us know, must have missed that one! A PR would be great, happy to accept if you want to submit a fix?

**Comment by ducksoupdev** [comment]
@cblades-tc I have fixed the issue and released v3.2.2
